blivet/devicefactory.py

```diff
@@ -137,9 +135,9 @@ def get_device_factory(blivet, device_type=DEVICE_TYPE_LVM, size=None, **kwargs)
                    DEVICE_TYPE_DISK: DeviceFactory}
 
     factory_class = class_table[device_type]
-    log.debug("instantiating %s: %s, %s, %s, %s", factory_class,
-              blivet, size, [d.name for d in disks], kwargs)
-    return factory_class(blivet, size, disks, **kwargs)
+    log.debug("instantiating %s: %s, %s, %s", factory_class,
+              blivet, [d.name for d in kwargs.get("disks", [])], kwargs)
```

Comment:
The original code removed the ``"disks"`` key from ``kwargs`` while the new version keeps it in. Was the old version wrong?